Sources/NIOHTTP1/HTTPUpgradeHandler.swift

```diff
@@ -60,6 +60,7 @@ public protocol HTTPProtocolUpgrader {
 /// requests that we choose to punt on it entirely and not allow it. As it happens this is mostly fine:
 /// the odds of someone needing to upgrade midway through the lifetime of a connection are very low.
 public class HTTPServerUpgradeHandler: ChannelInboundHandler {
+    
```

Comment:
Why are we adding this blank line?

Reply:
Will revert, left-over from some debugging Code that was there